README.md

```diff
@@ -22,6 +22,12 @@ Setting up your app to use ThunderBasics is a simple and quick process. You can
 - Add ThunderBasics.framework to your Embedded Binaries.
 - Wherever you want to use ThunderBasics use `import ThunderBasics` if you're using swift.
 
+## Building Binaries for Carthage
```

Comment:
Just above here, can you update the version in the install information to be `2.0.0` too? :D